src/main/scala/beam/analysis/cartraveltime/CarTripStatsFromPathTraversalEventHandler.scala

```diff
@@ -144,6 +148,7 @@ class CarTripStatsFromPathTraversalEventHandler(
     createIterationGraphForAverageSpeed(dataSet, iterationNumber, mode)
   }
 
+  //TODO: maybe collect all Iteration end events and then at the end compute an info for all iterations
```

Comment:
If you would do it only once in the end, then you would be able to review the graph and CSV files only in the very end. For production runs one iteration takes around 1 hour and usually we run 15-20 iterations, so all this time you won't have any idea what happens with simulation.